libs/tsconfig.json

```diff
@@ -0,0 +1,110 @@
+{
```

Comment:
this needed to be added back because unable to lint without it. shared folder build issues seem to be working with this